launch/launch/launcher.py

```diff
@@ -27,14 +27,12 @@
 
 
 class _TaskException(Exception):
-
     def __init__(self, task_descriptor_index, exception):
         self.task_descriptor_index = task_descriptor_index
         self.exception = exception
 
 
 class DefaultLauncher(object):
-
```

Comment:
PyCharm was complaining about PEP8 violations,
Looks like it isn't actually specified whether they should be there or not

I will add them back.